src/server/pps/cmds/cmds.go

```diff
@@ -192,15 +230,16 @@ $ {{alias}} -p foo -i bar@YYY`,
 			}
 			defer client.Close()
 
+			if raw {
+				e := cmdutil.Encoder(output, os.Stdout)
+				return client.ListJobFilterF(pipelineName, commits, history, true, filter, func(ji *ppsclient.JobInfo) error {
+					return e.EncodeProto(ji)
+				})
+			} else if output != "" {
+				return errors.New("cannot set --output (-o) without --raw")
+			}
+
 			return pager.Page(noPager, os.Stdout, func(w io.Writer) error {
-				if raw {
-					e := encoder(output)
-					return client.ListJobFilterF(pipelineName, commits, history, true, filter, func(ji *ppsclient.JobInfo) error {
```

Comment:
was this just broken with the pager before?

Reply:
I am not sure, I just wanted to make this more consistent - I think when outputting encoded data, we shouldn't use the pager (and I believe that is how other places in the codebase do it).